api/client.go

```diff
@@ -19,7 +19,7 @@ const (
 	authorization   = "Authorization"
 	cacheTTL        = "X-GH-CACHE-TTL"
 	graphqlFeatures = "GraphQL-Features"
-	mergeQueue      = "merge_queue"
+	features        = "merge_queue,branch_for_issue_api"
```

Comment:
The fields and mutations for this command are now public https://docs.github.com/en/graphql/reference/mutations#createlinkedbranch so I think we can remove the flag here 
```suggestion
	features        = "merge_queue"
```